### PR TITLE
Add mentoring exercise selector

### PR DESCRIPTION
**Pull request by kntsoriano**
https://github.com/exercism/v3-project-management/issues/187

### CODE REVIEW

File: app/helpers/react_components/student/request_mentoring_button.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          }
+        )
+      end
+
+      private
+      def request
+        {
+          endpoint: Exercism::Routes.api_solutions_url,
+          query: {
+            # TODO: Update API to accept arrays
```
Comment (kntsoriano) [inline app/helpers/react_components/student/request_mentoring_button.rb]: @iHiD The API currently doesn't accept arrays for `status` and `mentoring_status`.

File: app/helpers/react_components/student/request_mentoring_button.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+module ReactComponents
+  module Student
+    class RequestMentoringButton < ReactComponent
+      def to_s
+        super(
+          "student-request-mentoring-button",
+          {
+            request: request,
+            links: {
+              mentor_request: Exercism::Routes.new_track_exercise_mentor_request_path("$TRACK_SLUG", "$EXERCISE_SLUG")
```
Comment (ErikSchierboom) [inline app/helpers/react_components/student/request_mentoring_button.rb:16]: Nice. You're using the `$TRACK_SLUG` and `$EXERCISE_SLUG` arguments as placeholders in the URL :)

**Review by ErikSchierboom — APPROVED** [review]

**Comment by iHiD** [comment]
@kntsoriano Can you get this green pls? :) (You can leave the skipped test skipped)

**Comment by iHiD** [comment]
Great. This is nice :)

**Pull request merged**